miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/BoxService.java

```diff
@@ -5,6 +5,8 @@
 import java.util.List;
 import java.util.Map;
 
+import uk.ac.bbsrc.tgac.miso.core.data.Barcodable;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.ac.bbsrc.tgac.miso.core.data.Barcodable'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)